cmd/serve/grant.go

```diff
@@ -258,6 +258,8 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 		r.Mount("/v1/devicecheck", proxyRouter)
 		r.Mount("/v1/captchas", proxyRouter)
 		r.Mount("/v2/attestations/safetynet", proxyRouter)
+		// v3/captcha
+		r.Mount("/v3/captcha/", proxyRouter)
```

Comment:
Is there any difference between the change and
```suggestion
		r.Mount("/v3/captcha", proxyRouter)
```